modyn/trainer_server/internal/trainer/remote_downsamplers/deepcore_utils/README.md

```diff
@@ -0,0 +1,4 @@
+# DeepCore Utils
+The content of this folder is taken from DeepCore with minor adjustments.
```

Comment:
Neither conda nor pip is available; you can only clone the repo. Some functions (like OMP) are changed.

Reply:
Could you maybe in the README file explain what you changed? (such that if DeepCore ever gets updated we know how to integrate the updates)

Reply:
Ok done